### ISSUE TITLE
[sil_cameroon_qwerty] Touch Keyboard not using included font when downloaded.

### ISSUE THREAD

**Issue by erros84** [keymanapp/keyboards#515]
The font Andika Africa was successfully packaged with the keyboard, but it does not show up as the Keyboard font when added to iOS or Android within Keyman (+).

https://github.com/keymanapp/keyboards/blob/267222de529479c432d6e1200e63db106342498c/release/sil/sil_cameroon_qwerty/source/sil_cameroon_qwerty.keyman-touch-layout#L3

https://github.com/keymanapp/keyboards/blob/267222de529479c432d6e1200e63db106342498c/release/sil/sil_cameroon_qwerty/source/sil_cameroon_qwerty.kps#L39 

When using Developer's "Test Keyboard on Web" to install to the phone, the keyboard keys **do** use the font. Where did this break down when getting the file from the server?

**Comment by erros84** [comment]
@darcywong00 Any ideas? This is beyond me as it works properly locally.

**Comment by darcywong00** [comment]
The Android app queries https://api.keyman.com/cloud/3.0/languages/muc-Latn/sil_cameroon_qwerty?version=10.0.0&device=androidphone&languageidtype=bcp47 to download the sil_cameroon_qwerty keyboard (language muc-Latn in this example). This info is based on the sil_cameroon_qwerty.keyboard_info file.

I think there's two things to fix:

### sil_cameroon_qwerty.keyboard_info
* For each of the languages, you'll need to add a JSON object with the font information. Here's an example from yidgha.keyboard_info
```
  "languages": {
    "ydg-Arab": {
      "font": {
        "family": "Scheherazade",
        "source": "Scheherazade-Regular.ttf"
      }
    }
  },
```
This corresponds to http://s.keyman.com/font/deploy/Scheherazade-Regular.ttf

### Adding ANDIKAAFR-R.TTF to s.keyman.com
There's a gap in our infrastructure where fonts don't automatically get deployed to s.keyman.com when the keyboard gets merged into the repo.
* I can handle uploading ANDIKAAFR-R.TTF which will cover Android apps. I recommend Camel casing "AndikaAfr-R.ttf" for the filename & references.
* Somebody also needs to create an AndikaAfr-R.mobileconfig file so iOS can install the font.

**Comment by erros84** [comment]
Thanks @darcywong00 , @LornaSIL , and @DavidLRowe .  I just tested this on iOS. The font and OSK issues were solved through #539 , closing this issue.
